Description: Allow viewing changelogs of packages from Debian Ports repositories
Author: Manuel A. Fernandez Montecelo <manuel.montezelo@gmail.com>
Forwarded: no

--- a/src/generic/apt/pkg_changelog.cc
+++ b/src/generic/apt/pkg_changelog.cc
@@ -689,6 +689,7 @@
       if (!vf.File().end() &&
 	  vf.File().Origin() &&
 	  (string("Debian") == vf.File().Origin() ||
+	   string("Debian Ports") == vf.File().Origin() ||
 	   string("Debian Backports") == vf.File().Origin()))
 	{
 	  return true;
